CSharpSourceCode/TOW_Core.csproj

```diff
@@ -16,7 +16,7 @@
   </PropertyGroup>
   <!-- Bannerlord path Property-->
   <PropertyGroup>
-    <GameDir>C:\Program Files (x86)\Steam\steamapps\common\Mount &amp; Blade II Bannerlord</GameDir>
+    <GameDir>G:\SteamLibrary\steamapps\common\Mount &amp; Blade II Bannerlord</GameDir>
```

Comment:
remove this change from the PR pls, this is local to you.

Reply:
Ah, I am aware of this. I hadn't intended to start the review yet. I mainly wanted to to make it easier to look at some of the changes we had been discussing with Z3rca.
Didn't want to waste your time before I had finished. I am sorry. I should have made the initial comment clearer.

Thanks for the inputs though!